Source/Evolve/WarpXEvolve.cpp

```diff
@@ -165,6 +165,7 @@ WarpX::Evolve (int numsteps)
         if (do_electrostatic != ElectrostaticSolverAlgo::None)
         {
             const bool skip_deposition = true;
+            if (warpx_py_particleinjection) warpx_py_particleinjection();
```

Comment:
Since this is being merged upstream in a different branch I would revert this change, and then merge memaster into this branch (once the PR is complete and we have updated memaster).